pkgs/by-name/ko/komac/package.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, SystemConfiguration
+, testers
+, komac
+}:
+
+let
+  pname = "Komac";
+  version = "2.1.0";
+  src = fetchFromGitHub {
+    owner = "russellbanks";
+    repo = "Komac";
+    rev = "v${version}";
+    hash = "sha256-L8UYpNqjRyqf4hPQwD9LaXWu6jYaP34yTwTxcqg+e2U=";
+  };
+in
+rustPlatform.buildRustPackage {
+  inherit pname version src;
+
+  cargoHash = "sha256-S9y/SLs763CbLE9OiHYYMOaCrP74HSFiWcaukORLUis=";
+
+  buildInputs = lib.optionals stdenv.isDarwin [ SystemConfiguration ];
+
+  doCheck = true;
```

Comment:
i think default `doCheck` for `buildRustPackage` is `true`

Reply:
Removed the doCheck attribute in last commit ✅